templates.md

```diff
@@ -138,6 +138,16 @@ To overwrite a section entirely, you may use the `overwrite` statement:
 
 	{{-- This comment will not be in the rendered HTML --}}
 
+#### Service injection
+We can use `@inject` to resolve services from the service container.
+
+	@inject('html', 'HtmlBuilder')
```

Comment:
HtmlBuilder is not a class that's shipped with laravel by default, and is also wrong anyway - it's not the FQCN unless you've aliased it. Choose a different example please.